packages/opencensus-web-initial-load/karma.conf.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-const webpackConfig = require('./webpack/dev-package.config');
+const webpackConfig = require('./webpack.config');
```

Comment:
Would you be willing to keep a basic `karma.conf.js` file in `opencensus-web-all` in case we add tests there in the future?

Reply:
OK. Added.